introduction.tex

```diff
@@ -9,33 +9,72 @@ \chapter{Introduction}
 debugging tasks. However, in many scenarios, hardware support is essential.
 
 This document outlines a standard architecture for debug support
-on RISC-V platforms. This architecture allows a variety of implementations and
+on RISC-V hardware platforms. This architecture allows a variety of implementations and
 tradeoffs, which is complementary to the wide range of RISC-V implementations.
 At the same time, this specification defines common interfaces to
-allow debugging tools and components to target a variety of platforms based on the RISC-V ISA.
+allow debugging tools and components to target a variety of hardware
+platforms based on the RISC-V ISA.
 
 System designers may choose to add additional hardware debug support,
 but this specification defines a standard interface for common
 functionality.
 
 \section{Terminology}
 
-A \emph{platform} is a single integrated circuit consisting of one or more
-\emph{components}. Some components may be RISC-V cores, while others may have a different
-function. Typically they will all be connected to a single system bus.
-A single RISC-V core contains one or more hardware threads, called
-\emph{harts}.
-
-\emph{DXLEN} of a hart is its widest supported XLEN, ignoring the current value
-of \Fmxl in \Rmisa.
-
-A \emph{physical address} is directly usable on the system bus.
-
-A \emph{virtual address} is an address as a hart sees it. If there is address
-translation this may be different from the \emph{physical address}. If there is
-no translation then it will be the same.
-
-\subsection{Context}
+\begin{description}[style=nextline]
+    \item[AMO]
+        Atomic Memory Operation.
+    \item[BYPASS]
+        JTAG instruction that selects a single bit data register, also called BYPASS.
+    \item[component]
+        A RISC-V core, or other part of a hardware platform.
+        Typically all components will all be connected to a single system
+        bus.
+    \item[CSR]
+        Control and Status Register.
+    \item[DM]
+        Debug Module (see Section \ref{dm}).
+    \item[DMI]
+        Debug Module Interface (see Section \ref{dmi}).
+    \item[DR]
+        JTAG Data Register.
+    \item[DTM]
+        Debug Transport Module (see Section \ref{dtm}).
+    \item[DXLEN]
+        Debug XLEN, which is the widest XLEN a hart supports, ignoring the
+        current value of \Fmxl in \Rmisa.
+    \item[GPR]
+        General Purpose Register.
+    \item[hart]
+        A hardware thread in a RISC-V core.
+    \item[IDCODE]
+        32-bit Identification CODE, and a JTAG instruction that returns the IDCODE value.
+    \item[IR]
+        JTAG Instruction Register.
+    \item[JTAG]
+        Refers to work done by IEEE's Joint Test Action Group, described in
+        IEEE 1149.1.
+    \item[NAPOT]
+        Naturally Aligned Powers-Of-Two.
+    \item[NMI]
+        Non-Maskable Interrupt.
+    \item[hardware platform]
+        A single system consisting of one or more \emph{components}.
```

Comment:
The rest of this list is alphabetized.  When this item was changed from `platform` to `hardware platform`, it is now in the wrong place.